### PR TITLE
updated prometheus csv according to the integration into openshift

### PR DESCRIPTION
**Pull request by sichvoge**
As discussed, I updated the following fields:

- displayName -> Prometheus Operator
- description -> included Prometheus Operator info
- Maintainers and Provider -> Red Hat with openshift-operators@redhat.com
- Maturity -> beta

Changed: 
- Alert Manager -> Alertmanager

cc/ @brancz

### CODE REVIEW

**Comment by brancz** [comment]
lgtm

File: deploy/chart/catalog_resources/ocs/prometheusoperator.0.22.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -10,53 +10,47 @@ metadata:
...
-    [Read the complete guide to monitoring applications with the Prometheus Open Cloud Service](https://coreos.com/tectonic/docs/latest/alm/prometheus-ocs.html)
-
-    ### Supported Features
+    * **Target Services via Labels**: Automatically generate monitoring target configurations based on familiar Kubernetes label queries; no need to learn a Prometheus specific configuration language.
 
+    ### Other Supported Features
 
     **High availability**
 
-
```
Comment (robszumski) [inline deploy/chart/catalog_resources/ocs/prometheusoperator.0.22.2.clusterserviceversion.yaml]: These are getting jumbled up:

image

Comment (sichvoge) [reply deploy/chart/catalog_resources/ocs/prometheusoperator.0.22.2.clusterserviceversion.yaml]: MH. This is how it is displayed for me on OLM:

image

Comment (robszumski) [reply deploy/chart/catalog_resources/ocs/prometheusoperator.0.22.2.clusterserviceversion.yaml]: Hmm ok maybe I copy/pasted wrong

**Pull request merged**